### PR TITLE
Feature/email login

### PR DESCRIPTION
**Pull request by fungainyatanga**
Hey Guys
Anitha has an active pull request ahead of mine, **### so please do not merge yet** but I just want to make sure you have time to review
The best way is to pull the branch if you use postman here are the requests 
**Signup= http://localhost:5000/auth/signup**  
body{
	"email": "test123@gmail.com",
         "username": "test123",
	"password": "admin123"
}
Response: will just be a success user created. 

**Login=http://localhost:5000/auth/login**
body{
"email": "test123@gmail.com",
"password": "admin123"
}
Login will generate a token in the response
 You can create any user just wanted to give an example

### CODE REVIEW

File: src/auth/authHandlers.ts [hunk 1]
```diff
@@ -49,3 +49,30 @@ export const verifyToken = async ({ req, ...rest }) => {
 
 }
 
+export const validateData = (data: { email?: any; username?:any; password?: any }) => {
```
Comment (justmic007) [inline src/auth/authHandlers.ts:52]: I think email, username and password types should be `string` and not `any`. It should convert whichever data type supplied to a string

**Review by justmic007 — CHANGES_REQUESTED** [review]

**Comment by justmic007** [comment]
Bro,

I just tested with Postman. It works, though I left some comments on your
PR if you can look into it

File: src/server.ts [hunk 2]
```diff
@@ -48,6 +56,8 @@ app.get('/auth/linkedin/redirect', passport.authenticate('linkedin', {
   failureRedirect: '/',
 }), tokenGenerator);
 
+app.post('/signup', Signup);
```
[the remark this answers is not in the document]
Comment (fungainyatanga) [reply src/server.ts:59]: So I cant add redirect to login because page doesn't exist the Ui will have to handle once signup has occurred

**Review by justmic007 — APPROVED** [review]

**Comment by anithaamarnath** [comment]
It's working perfectly

**Review by anithaamarnath — APPROVED** [review]

**Pull request merged**